packages/desktop-electron/package.json

```diff
@@ -24,26 +24,22 @@
       "!node_modules/better-sqlite3/{benchmark,src,bin,docs,deps,build/Release/obj,build/Release/sqlite3.a,build/Release/test_extension.node}",
       "build"
     ],
-    "publish": {
-      "provider": "github",
-      "repo": "actual",
-      "owner": "actualbudget",
-      "vPrefixedTagName": true
-    },
+    "beforePack": "./build/beforePackHook.js",
     "mac": {
       "category": "public.app-category.finance",
       "icon": "icons/icon.icns",
       "hardenedRuntime": true,
       "gatekeeperAssess": false,
-      "artifactName": "${productName}-mac.${ext}",
+      "artifactName": "${productName}-mac-${arch}.${ext}",
       "notarize": {
         "teamId": "79ANZ983YF"
       },
       "target": [
         {
-          "target": "default",
+          "target": "dmg",
           "arch": [
-            "universal"
```

Comment:
No longer building for universal - it  might be possible to do this but it's more complicated. 

It would involve detecting "universal", building both arm64 and x64 better-sqlite and finding a way to point the app to the right arch depending on the users machine.